doc/ref_arch/openstack/chapters/chapter08.md

```diff
@@ -11,6 +11,10 @@
 <a name="8.1"></a>
 ## 8.1 Introduction
 
+The purpose of this chapter is to identify the gaps between what is required for automated deployment of VNFs on NFVI frameworks and framework offered by OpenStack.Once gaps are identified, the next step will be to propose a plan to address these gaps. 
+The most obvious way to address the gaps will be to propose a set of APIs in the upstream OpenStack community
+
+
```

Comment:
I like to cover the gap between real implementation and SDO / Opensource  , So we can propose it to ETSI NFV or/and Openstack ,...etc